tests/src/test/java/org/eclipse/hono/tests/DeviceRegistryHttpClient.java

```diff
@@ -272,14 +277,13 @@ private static String registrationWithoutIdUri(final String tenant) {
      *
      * @param tenantId The tenant to get information for.
      * @param expectedStatusCode The status code indicating a successful outcome.
-     * @return A future indicating the outcome of the operation. The future will contain the response payload if the
-     *         response contained the expected status code. Otherwise the future will fail with a
-     *         {@link org.eclipse.hono.client.ServiceInvocationException}.
```

Comment:
The removal of the `with a {@link org.eclipse.hono.client.ServiceInvocationException}` part is not consistent in this class yet - it's still in there for several methods.